### DISCUSSION TITLE
Closing tag

### DISCUSSION THREAD

**Discussion by ghost** [biomejs/biome#3695] [General]
I'm not sure what this formatting style is called, but I'd love an option to move the closing tag of an element with multiple attributes to the next line.

Currently, it looks like this:
```tsx
<Card.Flex
	justify='between'
	align='center'
	background={'bg-white'}
	padding={4}>
	// unrelated children
<Card.Flex/>
```

I would like it to behave like this:
```tsx
<Card.Flex
	justify='between'
	align='center'
	background={'bg-white'}
	padding={4}
> // <- this right here
	// unrelated children
<Card.Flex/>
```

**Comment by Conaclos** [comment]
We support this via the [`atterattributePosition`](https://biomejs.dev/reference/configuration/#formatterattributeposition) option.

**Comment by ghost** [comment]
Are you sure? That seems to be working only for attributes, i meant the closing tag.

```tsx
<div
	id={dropdownId}
	class={'relative w-fit dropdown-component'}> // this is how it is now
	{props.children}
</div>
```
```tsx
<div
	id={dropdownId}
	class={'relative w-fit dropdown-component'}
> // this is what im proposing
	{props.children}
</div>
```

**Reply by Conaclos** [reply]
[Playground](https://biomejs.dev/playground/?attributePosition=multiline&code=PABkAGkAdgAgAGkAZAA9AHsAZAByAG8AcABkAG8AdwBuAEkAZAB9ACAAYwBsAGEAcwBzAD0AewAnAHIAZQBsAGEAdABpAHYAZQAgAHcALQBmAGkAdAAgAGQAcgBvAHAAZABvAHcAbgAtAGMAbwBtAHAAbwBuAGUAbgB0ACcAfQA%2BAAoACQB7AHAAcgBvAHAAcwAuAGMAaABpAGwAZAByAGUAbgB9AAoAPAAvAGQAaQB2AD4A)?

**Reply by ghost** [reply]
Ok found the problem, it was `bracketSameLine`, thanks for the help!
